CHANGELOG.md

```diff
@@ -3,6 +3,148 @@
+
+  ```tsx
+  // To fully control
+  <DataGridPro
+    rowGroupingModel={rowGroupingModel}
+    onRowGroupingModel={newModel => setRowGroupingModel(newModel)}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+
+  // To initialize without controlling
+  <DataGridPro
+    initialState={{
+      rowGrouping: {
+        model: rowGroupingModel,
+      },
+    }}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+  ```
+
+  See the [documentation](https://mui.com/components/data-grid/group-pivot/#row-grouping) for more details.
+
+- ⚡ Add `is any of` filter operator (#2874) @alexfauquette
+
+  The new filter operator `is any of` allows the user to provide multiple values. It opens access to complex filtering pattern mixing `AND` and `OR` logic connectors, such as `status is any of filled or rejected, and currency is any of EUR or USD`.
+
+  <img src="https://user-images.githubusercontent.com/45398769/150486348-996a938f-db24-426f-bfe3-c06337f71807.gif" width="770" height="370">
+
+- ✨ Introduce a `maxWidth` property in `GridColDef` (#3550) @flaviendelangle
+
+  You can now limit the width of the flex columns and the resizable columns with the new `maxWidth` property on `GridColDef`.
+
+  ```tsx
+  const columns: GridColDef[] = [
+    { field: 'director', flex: 1, maxWidth: 200 }, // will take the free space up to 200px and will not be resizable above 200px
+    { field: 'year', maxWidth: 150 }, // will not be resizable above 150px
+  ]
+  ```
+
+- 🚀 Add component slots for a subset of used `@mui/material` components (#3490) @DanailH
+
+  To make the grid more flexible we added component slots for base `@mui/material` components that we use. Those component slots are prefixed with `Base` to differentiate them from the other grid specific components
+
+  For more information check the documentation [documentation](https://mui.com/api/data-grid/data-grid/#slots).
+
+- 🔥 Allow to pass `csvOptions` and `printOptions` to `toolbar` component prop (#3623) @flaviendelangle
+
+  ```tsx
+  const CustomDataGrid = (props: DataGridProps) => {
+    return (
+      <DataGrid {...props} componentsProps={{ toolbar: { csvOptions: { delimiter: ';' } } }} />
+    )
+  }
+  ```
+
+- 🙈 Add controlled behavior for the visible columns (#3554) @flaviendelangle
+
+  ```tsx
+  // To fully control
+  <DataGrid
+    columnVisibilityModel={columnVisibilityModel}
+    onColumnVisilibilityModelChange={newModel => setColumnVisibilityModel(newModel)}
+  />
+
+  // To initialize without controlling
+  <DataGrid
+    initialState={{
+      columns: {
+        columnVisibilityModel
+      }
+    }}
+  />
+  ```
+
+  See the [documentation](https://mui.com/components/data-grid/columns/#column-visibility) for more details.
+
+  The `hide` property from `GridColDef` still works but has been deprecated.
+
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.3.0` / `@mui/x-data-grid-pro@v5.3.0`
```

Comment:
@mui-org/x a thought on the changelog. With the ongoing work to correctly separate the files between the different packages and comments like https://twitter.com/Bifrost60541029/status/1485648490902528004 I wonder if it would make sense to more clearly isolate the changelog: what's MIT, Pro, Premium. This could help:

```suggestion
### `@mui/x-data-grid@v5.3.0`

### `@mui/x-data-grid-pro@v5.3.0`

Same changes as in `@mui/x-data-grid` with some extras:
```

Reply:
If we only put commits with [DataGridPro] tag in the `@mui/x-data-grid-pro@v5.3.0` section it will feel a bit empty. But it sounds good to me

Reply:
I agree it makes sense to separate based on the packages. But it's also a good point that it can put the pro and premium on a bad light.

I'm picturing the possibility of a piece of the changelog like this:

----------
### `@mui/x-data-grid-premium@vY.Z.X`
#### Same changes as in `@mui/x-data-grid` and `@mui/x-data-pro`
----------

I mean, depending how we frame it, from a not-so-understanding-customer POV, if that changelog pops up a few times, I think the perception of value might indeed decrease.

Do you think it will put some extra pressure on working for Pro and Premium?
My concern with it then is that it may be not a "good" kind of pressure (in the sense that it may push us off our explicit goals).

Reply:
Going back to why I have opened this thread. I think that we have a problem on being very transparent about which plan a change is released into. Take:

<img width="348" alt="Screenshot 2022-02-10 at 16 19 08" src="https://user-images.githubusercontent.com/3165635/153438078-79f44a3a-9f65-485e-bb69-705b37282088.png">

https://about.gitlab.com/releases/2022/01/22/gitlab-14-7-released/

It's clear. In our case, we use the "DataGrid" and "DataGridPro" conventions in the changelog. Which doesn't work for the highlights and might not work either for the Premium plan https://github.com/mui/mui-x/releases. 

> section it will feel a bit empty. 

@alexfauquette The way we solve this problem in https://github.com/mui/material-ui/releases is by not mentioning a package if there are no changes related to it. Now, since Pro extends MIT, the "Same changes as in" mention would probably be mandatory. 

> an put the pro and premium on a bad light.

@joserodolfofreitas If "bad light" is a reference to that we don't do changes in Pro/Premium, then I think that we should ignore this argument: this would be against our transparency value. Transparency is about confronting reality sooner. IMHO, really always win, we can't hide anything very long.

> Do you think it will put some extra pressure on working for Pro and Premium?

I didn't think about this aspect. From my perspective, no matter the grouping, we need to flag the plan the change is into. It has to be very clear. Happy to use any other system, it doesn't have to be with a group.

Reply:
I think another problem here is that `DataGrid` doesn't imply it's free.
For comparison, in GitLab it literally says FREE.
Maybe we should do something like this:

---
### `@mui/x-data-grid@vX.Y.Z` ([Community plan](https://mui.com/components/data-grid/getting-started/#plans))

#### Changes

- 

### `@mui/x-data-grid-pro@vX.Y.Z` ([Pro plan](https://mui.com/components/data-grid/getting-started/#plans))
#### Changes

Same changes as in `@mui/x-data-grid` with some extras:

- 

### `@mui/x-data-grid-premium@vX.Y.Z` ([Premium plan](https://mui.com/components/data-grid/getting-started/#plans))
#### Changes

Same changes as in `@mui/x-data-grid` and `@mui/x-data-grid-pro` with some extras:

-


...

Reply:
Could we move the discussion to an issue or to Notion
Discussing on closed PR is not a good idea for me, it's very easy to miss it.

Reply:
I added this discussion as a section on our "Premium decisions" page.
https://www.notion.so/mui-org/Shipping-premium-plan-782c6648a4ae4a0491a3d17d65087546